.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup
         uses: actions/setup-node@v2
         with:
-          node-version: 16
+          node-version: 19.3
```

Comment:
I'm seeing that github actions only supports LTS versions, so please change to 18 that it is the latest LTS version

Reply:
downgraded version to 18